solana/Dockerfile.wasm

```diff
@@ -1,18 +1,21 @@
 # syntax=docker.io/docker/dockerfile:1.3@sha256:42399d4635eddd7a9b8a24be879d2f9a930d0ed040a61324cfdf59ef1357b3b2
 FROM docker.io/library/rust:1.49@sha256:a50165ea96983c21832578afb1c8c028674c965bc1ed43b607871b1f362e06a5 AS build
 
-RUN apt-get update && apt-get install -y libssl-dev libudev-dev pkg-config zlib1g-dev llvm clang
-RUN rustup component add rustfmt
-RUN rustup default nightly-2022-01-02
-
 WORKDIR /usr/src/bridge
 
 # Support additional root CAs
-COPY devnet_setup.sh cert.pem* /certs/
+COPY cert.pem* /certs/
 # Debian
 RUN if [ -e /certs/cert.pem ]; then cp /certs/cert.pem /etc/ssl/certs/ca-certificates.crt; fi
 
-RUN cargo install wasm-pack --vers 0.9.1
+COPY rust-toolchain .
+
+# Install wasm-pack binary, saves about 2 minutes relative to building it from
+# source
+ARG WASM_PACK_VER=v0.9.1
```

Comment:
I'm not sure we want to trust another binary artifact here for security reasons

Reply:
verified that they produce identical wasms and added a checksum

Reply:
@claudijd how do you feel about this?

Reply:
Seems ok to me so long as we do a manual build from a git checkout to verify the checksum before uprevving this.

Reply:
I have done that manual bit, if that counts